src/Control/SAMLController.php

```diff
@@ -52,19 +51,19 @@ public function index()
      * if not existent), with the user already logged in. Login triggers memberLoggedIn hooks, which allows
      * LDAP side of this module to finish off loading Member data.
      *
-     * @throws OneLogin_Saml2_Error
+     * @throws OneLogin\Saml2\Error
```

Comment:
please just use `Error` once you've added the `use` statement (see other comments)